hardhat.config.ts

```diff
@@ -9,8 +9,8 @@ import 'hardhat-deploy';
 import 'hardhat-gas-reporter';
 import { HardhatUserConfig, task } from 'hardhat/config';
 
-import '@matterlabs/hardhat-zksync-deploy';
-import '@matterlabs/hardhat-zksync-solc';
+// import '@matterlabs/hardhat-zksync-deploy';
```

Comment:
May want to think about flagging this somehow so we don't have to keep commenting/uncommenting?

Reply:
Adding a example env with a flag for deploying to zksync